index.html

```diff
@@ -36,26 +42,24 @@ <h4 class="headline">LET’S CONNECT</h4>
       <li><a href="#"><img src="images/Vector(1).svg" alt="gitbubicon" /></a></li>
       <li><a href="#"><img src="images/Vector.svg" alt="thumb" /></a></li>
     </ul>
-  </section>
+  </div>
+</section>
 
-
-
-
-
-
-  
-  <section>
-    <div class="cards-container">
-      <ul class="cards-list flex-center-culomn">
-        <li class="card-item flex-center-culomn">
+<section>
+  <div class="cards-container">
+    <ul class="cards-list flex-center-culomn">
+      <li class="card-item flex-center-culomn">
+        <div class="img-container">
           <img class="card-image" src="images/SnapshootPortfolio.jpg" alt="Portfolio Portfolio">
+        </div>
+        <div class="right-container">
           <div class="list-conatiner">
             <h2>Tonic</h2>
-            <div class="list">
+             <div class="list">
               <span class="list-item item1">Canopy</span>
-              <span class="list-item">back end Dev</span>
-              <span class="list-item">2015</span>
-            </div>
+              <span class="list-item"><img src="images/Counter.jpg" alt="Counter">back end Dev</span>
+              <span class="list-item"><img src="images/Counter.jpg" alt="Counter">2015</span>
+             </div>
```

Comment:
- [ ] The content inside the cards looks a bit congested, please consider adding more podding between the content sections and set the font color of `<p>` tag to `#344563`: 


Your design  | Figma design
------------- | -------------
<img width="337" alt="Screenshot 2022-04-01 at 12 28 57" src="https://user-images.githubusercontent.com/77038610/161225910-995746d7-2ed9-411a-b1bf-27e3cdacbd7a.png"> |  <img width="372" alt="Screenshot 2022-04-01 at 12 28 35" src="https://user-images.githubusercontent.com/77038610/161225851-302cbea2-88df-43b0-8035-b9610ff76278.png">